plenum/test/freshness/test_freshness_batch_updates_last_ordered.py

```diff
@@ -73,8 +73,8 @@ def test_freshness_batch_updates_last_ordered_non_primary(looper, txnPoolNodeSet
 
     restart_node(looper, txnPoolNodeSet, txnPoolNodeSet[1], tconf, tdir, allPluginsPath)
 
-    assert txnPoolNodeSet[0].master_replica.txnRootHash(DOMAIN_LEDGER_ID) == \
-           txnPoolNodeSet[1].master_replica.txnRootHash(DOMAIN_LEDGER_ID)
+    assert txnPoolNodeSet[0].master_replica.node.db_manager.get_txn_root_hash(DOMAIN_LEDGER_ID) == \
+           txnPoolNodeSet[1].master_replica.node.db_manager.get_txn_root_hash(DOMAIN_LEDGER_ID)
```

Comment:
I suspect this can be simplified just to `txnPoolNodeSet[x].db_manager.get_txn_root_hash`